<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyTDownloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Video-URL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="video_url_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>461</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="download_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QComboBox" name="quality_comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QListWidget" name="completed_downloads_listWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>461</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="current_download_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>461</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Aktuell:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Qualit√§t</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
